package/ios/Core/CameraSession.swift

```diff
@@ -121,10 +136,12 @@ final class CameraSession: NSObject, AVCaptureVideoDataOutputSampleBufferDelegat
         try lambda(config)
       } catch CameraConfiguration.AbortThrow.abort {
         // call has been aborted and changes shall be discarded
+        completionBlock?()
```

Comment:
instead of writing that 3 times, put it in a `defer { ... }` block so we dont forget it when we add a fourth exit point